src/nvapi_d3d.cpp

```diff
@@ -6,14 +6,34 @@ extern "C" {
 
     NvAPI_Status __cdecl NvAPI_D3D_GetObjectHandleForResource(IUnknown *pDevice, IUnknown *pResource, NVDX_ObjectHandle *pHandle) {
         static bool alreadyLogged = false;
-        return NoImplementation("NvAPI_D3D_GetObjectHandleForResource", alreadyLogged);
+        constexpr auto n = __FUNCTION__;
+
+        // Fake-implement with a dumb passthrough, though no other NvAPI entry points
+        // we're likely to implement should care about the actual handle value.
+        *pHandle = (NVDX_ObjectHandle)pResource;
+
+        return Ok(n, alreadyLogged);
```

Comment:
Since `n` is used only once, we can omit the declaration and use the value directly like `return Ok("NvAPI_D3D_GetObjectHandleForResource", alreadyLogged);`. (See comment above about `__FUNCTION__`)